managed/services/agents/vmagent_test.go

```diff
@@ -0,0 +1,24 @@
+package agents
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMaxScrapeSize(t *testing.T) {
+	t.Run("by default 64MiB", func(t *testing.T) {
```

Comment:
Maybe default value should be exported in managed/services/agents/vmagent.go and used here?
Every change of default value will cause this test fail. 
What do you think?